data/fixtures/recorded/actions/snippets/snipFunkToThis.yml

```diff
@@ -1,42 +0,0 @@
-languageId: typescript
```

Comment:
This is being temporarily removed but tracked in https://github.com/cursorless-dev/cursorless/issues/2353. "Context specific scope matching (i.e. method inside of a class versus outside)"